README.md

```diff
@@ -91,12 +91,39 @@ public class MainApplication extends Application implements ReactApplication {
 
 ### Assets generation
 
-In order to speed up the setup, we provide a **CLI** to resize assets, create the Android Drawable XML file and the iOS Storyboard file automatically ✨.
+**NEW** React-Native BootSplash provides a react-native plugin to allow contextual creation of your bootsplash
+
+```bash
+# Noninteractive deployment
+$ react-native init-bootsplash --iconPath=myicon.png
```

Comment:
Not sure about providing two different CLI entry points, it's quite confusing. I would rather use something like [meow](https://github.com/sindresorhus/meow) and keep a `yarn / npx` only workflow.

What about:

```sh
# interactive mode
> yarn generate-bootsplash

# display the help
> yarn generate-bootsplash --help

# the only required arg is the original logo filename
> yarn generate-bootsplash bootsplash.png

# optional args use kebab case, --project-path can be useful for weird monorepo setups
> yarn generate-bootsplash bootsplash.png --project-path ./packages/mobile --assets-path ./assets --background-color "#fff" --logo-width 100
```

Reply:
I agree on the virtue of having having one first-class entry point. I've been a fan of using the react-native CLI plugin structure for my various RN tools. For consuming devs, this means one point of accessing their RN utilities. Using `react-native --help` inside a project to get the full universe is kind of magical, and reduces cognitive load in such an extensible ecosystem, like having all your various tools in one toolbox. I think that after a couple more iterations (dark mode, automating the deeper links) this project could and should be the main way RN devs make and manage their launch screens. 

For toolmakers, instead of rolling our own argument managed CLI (using meow, gluegun, commander, etc), we get to just make use of RN's via the `commands` member of `react-native.config.js`. And it's not just about me: a future RN dev who wants to further extend the functionality of this plugin gets to play in a familiar sandbox with less work. 

I adjusted the documentation to characterize the `react-native generate-bootsplash` as primary and `yarn generate-bootsplash` as legacy. Let me know what you think.

Reply:
A quick note on the preceding - I just found myself mis-typing a command for a plugin in react-native. It was super-helpful in suggesting the thing I likely meant. 

(Confession: the mistake was when I was applying one of my own plugins: [react-native-usage-description](https://github.com/rhdeck/react-native-usage-description))

Reply:
Does the `react-native` CLI plugin allow interactive mode using `prompts`? If it supports it , I don't see the point of providing both in parallel.

As the current CLI tool could not run programmatically, a simple warning that said `This method is deprecated, use react-native generate-bootsplash instead` seems close enough.

Reply:
It works as you hope: running react-native generate-bootsplash introduces the same workflow as you had previously with the additional option of adding the bootsplash to the pbxproj and plist files.

I updated the README in the PR to make this all clearer.

Reply:
Awesome, let's do it this way. A deprecation message on `yarn generation-bootsplash` will be good enough, we keep one way do it (I will remove it later if [this fix](https://twitter.com/zoontek/status/1251873824028733441) get merged into RN - a breaking change will be needed, but it would improve Android DX a lot).

Could you rebase your PR? 🙂

Reply:
Rebased